### PR TITLE
[4.0] Blog sampledata text

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #31596 .

### Summary of Changes
Language texts in Blog Sample Data come from version 3.x. 
While expanding the Plugin for Cassiopiea, texts were changed not much so that translators have not too much work with changing the texts.
This means: All inconsitencies in texts are there since 3.x

This PR fixes a few things, where changes were made in 4.0 
- The Admin menu is no longer in position top a. The position now is not indicated directly, so we are more flexible .
- The description of Modules is re-added from 3.9
- The new tag Joomla V4 is now Joomla 4.

Re-writing the texts and adding new information needs some love and a native speaker. 

### Testing Instructions
Install a fresh Joomla and then install blog sample data.

### Actual result BEFORE applying this Pull Request
In some texts the position top menu was used. This is changed to a neutral name.  The Tag name Joomla V4 is Joomla 4.

### Expected result AFTER applying this Pull Request
The name "top menu" is changed to a neutral name.  The Tag name Joomla V4 is Joomla 4.

### Documentation Changes Required
no

### CODE REVIEW

**Comment by Bakual** [comment]
@brianteeman Can you please check the strings to make sure it is proper english :-)

**Comment by infograf768** [comment]
Can you also change the desc?
from
`PLG_SAMPLEDATA_BLOG_OVERVIEW_DESC="Sample data which will set up a blog site.<br>If the site is multilingual, the data will be tagged to the active backend language."`

to something like
`PLG_SAMPLEDATA_BLOG_OVERVIEW_DESC="Sample data which will set up a blog site including workflows for Cassiopea default template.<br>If the site is multilingual, the data will be tagged to the active backend language."`

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 1]
```diff
@@ -3,14 +3,14 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_SAMPLEDATA_BLOG="Sample Data - Blog"
-PLG_SAMPLEDATA_BLOG_OVERVIEW_DESC="Sample data which will set up a blog site.<br>If the site is multilingual, the data will be tagged to the active backend language."
+PLG_SAMPLEDATA_BLOG="Sample Data for Cassiopeia (Blog)."
```
Comment (Quy) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG="Sample Data for Cassiopeia (Blog)"
```

**Comment by adj9** [comment]
I have not tested this item.<br /><br />The menu works correctly, it's not just on the top page.

I can't find the new tag!

Minus the tag you can confirm the PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31604">issues.joomla.org/tracker/joomla-cms/31604</a>.</sub>

**Comment by chmst** [comment]
added unique text for issue #31642

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 55e2c1107b99e4fd2aa9c7a4f6416c51468388f0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31604">issues.joomla.org/tracker/joomla-cms/31604</a>.</sub>

**Comment by richard67** [comment]
> added unique text for issue #31642

@chmst The new menu description is "The horizontal dropdown menu". But the menu is only horizontal when shown in the header area, e.g. on module position "menu". On other positions it will be vertical, unless the menu class "menu-horizontal" is set in the module's advanced option, which is as far as I remember not the case after having installed Blog Sample Data.


Suggestion: Either change the description to "The dropdown menu" or "The accessible dropdown menu" or something like that, or set that menu class for the module in Blog Sample Data. I'd prefer the first solution, but maybe others prefer the 2nd because it could serve as an example for using that menu class.

**Comment by infograf768** [comment]
@richard67 
Is it really important as it is a sample data specifically aimed at Cassiopea and will be deleted soon or later?

**Comment by richard67** [comment]
@infograf768 Well, it could be confusing if someone installs Blog Sample Data and then moves the menu to another place. But it's not really important, it was just a suggestion by review, not required.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 55e2c1107b99e4fd2aa9c7a4f6416c51468388f0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31604">issues.joomla.org/tracker/joomla-cms/31604</a>.</sub>

**Comment by chmst** [comment]
@richard67 I would prefer to let the name as it is, because it describes cassiopeia after the installation of blog sample data.
If a user wants to move the module or change the class - he as well can change the description.

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31604">issues.joomla.org/tracker/joomla-cms/31604</a>.</sub>

**Comment by HLeithner** [comment]
thanks

**Pull request merged**